{"name": "Проверка получения ошибки авторизации курьера при вводе невалидных данных", "status": "passed", "parameters": [{"name": "invalid_data", "value": "{'login': 'NatashaLaktionova14', 'password': '1222'}"}], "start": 1730055278254, "stop": 1730055278325, "uuid": "f66ce5dd-008e-4d5d-bc97-7fef2f5e52f0", "historyId": "77718e1b33f0a633906c339ccff0eca9", "testCaseId": "e4e7857049db4d93f50eb72f9173712d", "fullName": "tests.test_login_courier.Test_login_courier#test_courier_login_nonexistent_data_not_found", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "Test_login_courier"}, {"name": "host", "value": "MacBook-Air-Natasha.local"}, {"name": "thread", "value": "80125-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}